README.md

```diff
@@ -351,4 +351,54 @@ all possible options,
 we have parameterized the top-level `settings.gradle` file.
 The `settings.gradle` file determines the scope of the build.
 Connectors requiring a driver are included in the overall build *only if* the associated driver location environment variable is present.
-Otherwise, the connector is ignored for the build.
\ No newline at end of file
+Otherwise, the connector is ignored for the build.
+
+## Building Docker Images
+
+The connectors in this repository contain `gradle` tasks that can be used to build Docker Images for each connector and 
+push those images to the developer's Docker Registry. In order to use these tasks, the developer must include the 
+following configuration option in their `gradle.properties` file, along with some optional parameters:
+
+*   `dockerRegistry`: Required. The name of the registry to which the image should be pushed (i.e. `docker.io`,
+`quay.io`, etc.).
+*   `pathToRepo`: Required. The path to the docker repository. This is typically the `namespace` portion of registry
```

Comment:
It's fine & we can fix later.  I would remove the _must begin & end with '/'_ stuff and add as necessary.  But we can fix this later. This one's churned enough.